materials/outline.md

```diff
@@ -0,0 +1,38 @@
+# Entitlement Justice and Measures of Algorithmic Fairness
+
+## Introduction
+ - Summarize importance of algorithmic fairness measures
+ - Introduce the concept of entitlement justice
+ - Introduce argument
+   - Entitlement justice is important in algorithmic fairness, but historically neglected in discussions of fairness measures
+   - (Some/certain) counterfactuals measures emphasize entitlement justice, resolving this neglect
+
+## Entitlement Justice
+ - Present Nozick's theory of entitlement justice
+ - Aknowledge known issues, but extract key points of relevance
```

Comment:
Key point I want to get at in this section is the idea of one being entitled to the holdings that they earn. This should carry the argument that people are entitled to, say, admissions or loans in the case when they've earned them.